protocol/0044-LIME-lp_mechanics.md

```diff
@@ -341,11 +341,19 @@ In the case of spot markets it will be transferred into the network treasury for
   - LP can cancel their commitment without incurring penalties (<a name="0044-LIME-053" href="#0044-LIME-053">0044-LIME-053</a>)
 - For a market that is in continuous trading and a single LP has committed liquidity:
   - The LP can cancel their commitment at any time (though this may involve incurring a penalty) (<a name="0044-LIME-060" href="#0044-LIME-060">0044-LIME-060</a>) for spot (<a name="0044-LIME-056" href="#0044-LIME-056">0044-LIME-056</a>)
+  - The LP can increase their commitment at any time without incurring fee (<a name="0044-LIME-061" href="#0044-LIME-061">0044-LIME-061</a>) for spot (<a name="0044-LIME-062" href="#0044-LIME-062">0044-LIME-062</a>)
+  - When LP increases their commitment higher than their account balance are rejected and old provision persists (<a name="0044-LIME-063" href="#0044-LIME-063">0044-LIME-063</a>) for spot (<a name="0044-LIME-064" href="#0044-LIME-064">0044-LIME-064</a>)
+  - When LP decreases their commitment are not charged (<a name="0044-LIME-065" href="#0044-LIME-065">0044-LIME-065</a>) for spot (<a name="0044-LIME-066" href="#0044-LIME-066">0044-LIME-066</a>)
+  - When LP decreases their commitment to zero are rejected (<a name="0044-LIME-067" href="#0044-LIME-067">0044-LIME-067</a>) for spot (<a name="0044-LIME-068" href="#0044-LIME-068">0044-LIME-068</a>)
+  - When LP amends the Fee Factor greater than `market.liquidityV2.maximumLiquidityFeeFactorLevel` are rejected (<a name="0044-LIME-070" href="#0044-LIME-070">0044-LIME-070</a>) for spot (<a name="0044-LIME-071" href="#0044-LIME-071">0044-LIME-071</a>)
```

Comment:
This should only be temporary core naming for the parameter. Once everything is on SLA the `V2` should go away. Specs shouldn't use `V2`.